src/store/auth.js

```diff
@@ -73,50 +84,46 @@ export default {
           emailIsSent: false,
           notFoundMurren: false,
         }
-        const {data} = await axios.post('/murren/reset_password/', payload)
 
-        if (data.email_sent_successfully) {
-          results.emailIsSent = true
-        }
+        const {data} = await axios.post('/auth/users/reset_password/', payload)
 
-        if (data.error_on_backend && data.error_text === 'Murren matching query does not exist.') {
-          results.notFoundMurren = true
+        if (data.status === 204) {
+          results.emailIsSent = true
         }
```

Comment:
Destructuring is applied here, therefore in the data variable there is only a backend data, if you need to get status you can go in several ways

```
const {data, status} = await axios.post()
status === 204

const {data, ...response} = await axios.post()
response.status === 204

const response = await axios.post()
response.status === 204
```

Reply:
done